python/packages/autogen-ext/src/autogen_ext/tools/graphrag/_global_search.py

```diff
@@ -39,12 +38,54 @@ class GlobalSearchTool(BaseTool[GlobalSearchToolArgs, GlobalSearchToolReturn]):
 
     This tool allows you to perform semantic search over a corpus of documents using the GraphRAG framework.
     The search combines graph-based document relationships with semantic embeddings to find relevant information.
+    Example usage with AssistantAgent:
+
+    .. code-block:: python
+
+        import asyncio
+        from autogen_ext.models.openai import AzureOpenAIChatCompletionClient
+        from autogen_ext.tools.graphrag import GlobalSearchTool
+        from azure.identity import DefaultAzureCredential, get_bearer_token_provider
+        from autogen_agentchat.agents import AssistantAgent
+
+
+        async def main():
+            # Initialize the OpenAI client
+            openai_client = OpenAIChatCompletionClient(
+                model="gpt-4o-mini",
+                api_key="<api-key>",
+            )
+
+            # Set up global search tool
+            global_tool = GlobalSearchTool.from_settings(settings_path="./settings.yaml")
```

Comment:
Is there a minimum example for the `settings.yaml` that can work with a few text documents?

Reply:
We would like to have a dev to just copy and run the example in a script and it will work out of box. 

So let's also include the installation instructions:

```
pip install "autogen-agentchat==0.4.0.dev12" "autogen-ext[graphrag]==0.4.0.dev12"
```